analysis/deep2_r23_o32/r_temp_calcul.py

```diff
@@ -177,17 +177,20 @@ def run_function(fitspath, dataset, verification_table, dustatt=False):
         Three_Beta = (OIII5007 * (1 + 1 / 3.1)) / HBETA
 
     # Raw Data
-    R_value = R_calculation(OIII4363, OIII5007, EBV)
-    T_e = temp_calculation(R_value)
-    metal_dict = metallicity_calculation(T_e, Two_Beta, Three_Beta)
+    R_value = R_calculation(OIII4363, OIII5007)
+    T_e = temp_calculation(R_value, EBV=None)
+    # Need to go back and make det3 equal to array because we have a validation table created
+    metal_dict = metallicity_calculation(T_e, Two_Beta, Three_Beta, EBV=None, det3=None)
```

Comment:
Ultimately calling MSC flux_ratios will generate tables with T_e and metallicity.  This will be implemented in a forthcoming version.

Reply:
Noted

Reply:
noted